client/src/components/graphs/LikertScale.js

```diff
@@ -4,14 +4,22 @@ import PropTypes from "prop-types"
 import Text from "react-svg-text"
 import useDimensions from "react-use-dimensions"
 
-const LikertScale = ({ onChange, value, levels, width, height, readonly }) => {
+const LikertScale = ({
+  onChange,
+  value,
+  values,
+  levels,
+  width,
+  height,
+  readonly
+}) => {
   const cursorRef = useRef(null)
   const axisRef = useRef(null)
   const [containerRef, containerBox] = useDimensions()
   const containerHeight = containerBox.height || 0
   const containerWidth = containerBox.width || 0
   const containerX = containerBox.x || 0
-  const MARGIN = 20
+  const MARGIN = 13
```

Comment:
This seems to have been changed from `20` to `5` and then to `13`. Looks like a magic number to me. ☺

Reply:
It was the smallest number for which the whole widget is still visible, including the start pointer containing the 0.0.

Reply:
In any browser, with any font size and any zoom level etc.? ☺